dojo/importers/utils.py

```diff
@@ -69,7 +69,7 @@ def update_import_history(type, active, verified, tags, minimum_severity, endpoi
 
     # tags=tags TODO no tags field in api for reimport it seems
     if endpoints_to_add:
-        import_settings['endpoints'] = endpoints_to_add
+        import_settings['endpoints'] = [str(endpoint) for endpoint in endpoints_to_add]
```

Comment:
@valentijnscholten do we really what to save the string representation of the endpoint or it is better to save the id of the endpoint? 
I prefer the id.

Reply:
maybe both? To  me the id says nothing and I'd like to see the actual URI

Reply:
Maybe we save the id in the database but display the string representation in the ui?

Reply:
If we add ids in the json, we have to lots of database lookups to display the URI in the UI. Maybe we should move it out of the settings and as its own field / relation on `Test_Import`.
For now maybe merge only the bugfix so it can be part of the release on Tuesday? I fixed the flake8 violation.